### ISSUE TITLE
What is the correct learning rate for training on a single GPU?

### ISSUE THREAD

**Issue by AndreaPi** [open-mmlab/mmdetection#6130]
The documentation is a bit confusing: here it seems that for training on a single GPU, just using the `tools/train.py` script is sufficient

https://mmdetection.readthedocs.io/en/latest/1_exist_data_model.html?highlight=single%20gpu#training-on-a-single-gpu

However, here it seems that I have to change the learning rate before training on a single GPU:

https://mmdetection.readthedocs.io/en/latest/1_exist_data_model.html?highlight=single%20gpu#train-predefined-models-on-standard-datasets

So, what's the right thing to do? Suppose I still use 2 images for GPU: I have a 32 Gb GPU, so I guess might use more, but let's consider 2 samples to fix ideas. Should I add this line to my config file?

`optimizer = dict(type='SGD', lr=0.0025, momentum=0.9, weight_decay=0.0001)
`
To avoid having to add always the same line to different config files, can I put the line in a file `lr_for_batchsize_2.py` and then append it to my `_base` variable? In other words, I will have this instruction at the beginning of my config file:

`_base_ = ['../mask_rcnn/mask_rcnn_r50_fpn_1x_coco.py', '/path/to/lr_for_batchsize_2.py']
`

Will this work?

**Comment by ZwwWayne** [comment]
Yes, the original setting works for a total batch size of 8x2 with 8 GPUS x 2 samplers_per_gpu.
